Formula/gl2ps.rb

```diff
@@ -58,6 +58,7 @@ def install
     EOS
     system ENV.cc, "-L#{lib}", "-lgl2ps", "-framework", "OpenGL", "-framework", "GLUT", "-framework", "Cocoa", "test.c", "-o", "test"
     system "./test"
-    assert_predicate testpath/"test.eps", :exist? && File.size("test.eps") > 0
+    assert_predicate testpath/"test.eps", :exist?
+    assert_predicate File.size("test.eps"), :positive?
```

Comment:
so :crying_cat_face:

Reply:
Note I'm pretty sure the previous test wouldn't even work as expected if that makes things any better 😉

Reply:
I was leaving this one unfixed as a protest.

Reply:
The 🚓 sent in the 🐴 to your protest.

Reply:
:mask: